js/web/_helper/js/_helper.js

```diff
@@ -166,45 +166,45 @@ let HTML = {
 		}
 
 		// es gibt gespeicherte Koordinaten
-		if(cords !== null){
+		if(cords){
 			let c = cords.split('|');
 			div.offset({ top: Math.min(c[0], window.innerHeight - 50), left: Math.min(c[1], window.innerWidth - 100) });  // Verhindere, dass Fenster außerhalb plaziert werden
 		}
 
 		// Ein Link zu einer Seite
-		if(args['ask'] !== undefined){
+		if(args['ask']){
 			div.find(title).after( $('<span />').addClass('window-ask').attr('data-url', args['ask']) );
 		}
 
 		// wenn Box im DOM, verfeinern
 		$('body').append(div).promise().done(function() {
-			if(args['auto_close'] !== undefined){
+			if(args['auto_close']){
 				$('body').on('click', '#' + args['id'] + 'close', function(){
 					$('#' + args['id']).fadeToggle('fast', function(){
 						$(this).remove();
 					});
 				});
 			}
 
-			if(args['ask'] !== undefined) {
+			if(args['ask']) {
 				$('body').on('click', '.window-ask', function() {
 					window.open( $(this).data('url'), '_blank');
 				});
 			}
 
-			if(args['dragdrop'] !== undefined) {
+			if(args['dragdrop']) {
 				HTML.DragBox(document.getElementById(args['id']), args['saveCords']);
 			}
 
-			if(args['resize'] !== undefined){
+			if(args['resize']) {
 				HTML.Resizeable(args['id']);
 			}
 
-			if(args['minimize'] !== undefined){
+			if(args['minimize']) {
```

Comment:
@dsiekiera pls don't use strict equals with undefined, null etc. because of this !== undefined next code is not working:
```
'minimize': false
```
instead of `if (foo === undefined)` just use `if (foo)`. Only if you REALLY need differentiate null, undefined, 0, '' than make sense to use strict equals. 99% code usually dont need it

I checked all places where used minimize: false, and changes to minimize: true... because I think evryone copypate wrong code when creating Box. Same comment for rest, dragdrop, etc